### PR TITLE
[FIX] website: do not allow to create/edit views

### PR DESCRIPTION
**Pull request by sswapnesh**
Before this commit, 
User was allowed to create new records from form view through
Graph view' action. Since we do not have a way to prevent opening 
form view from Graph and also we should not Introduce new form view
on stable version.

With this commit,
we are disabling create/edit/copy from action.

PS: I don't think it good idea to add disable_lining on Graph here as we still need to view record from Graph.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by sswapnesh** [comment]
Ping @rdeodoo

File: addons/website/views/website_visitor_views.xml
Comment (rdeodoo) [inline addons/website/views/website_visitor_views.xml:324]:
```suggestion
        <field name="context">{'search_default_type_url': 1, 'create': False, 'edit': False, 'copy': False}</field>
```
(yes also fix the existing 'typo' while editing this line)

**Review by rdeodoo — APPROVED** [review]
Not really a big deal imo.. but since we have the PR
LGTM once you change the line as I suggested

File: addons/website/views/website_visitor_views.xml
Comment (sswapnesh) [reply addons/website/views/website_visitor_views.xml:324]: > (yes also fix the existing 'typo' while editing this line)

Which ?

**Comment by JKE-be** [comment]
@robodoo delegate=rdeodoo

File: addons/website/views/website_visitor_views.xml
Comment (rdeodoo) [reply addons/website/views/website_visitor_views.xml]: The missing space after `'search_default_type_url'` :+1:

**Comment by rdeodoo** [comment]
@robodoo r+

**Comment by rdeodoo** [comment]
@sswapnesh can you rebase please? I guess this one is too old and it makes the build upgrade fails

**Comment by sswapnesh** [comment]
Ping @rdeodoo

**Comment by JKE-be** [comment]
@sswapnesh, seems like we miss your first rebase.

Since there are 25 days between your rebase and the ping, the merge is again impossible.
Please, rebase a ping us again. Since you don't provide rights on your PR, we cannot do it ourself.

**Comment by sswapnesh** [comment]
Rebase done.

>  Since you don't provide rights on your PR, we cannot do it ourself.

Allow edits by maintainers in Turned on.

**Comment by rdeodoo** [comment]
@robodoo r+

**Pull request closed without merging**
